Dockerfile

```diff
@@ -8,8 +8,7 @@ ARG NEXT_TELEMETRY_DISABLED=1
 COPY . .
 
 RUN yarn install --frozen-lockfile
+RUN yarn compile
```

Comment:
I'm not sure what are you trying to do but local dev environment is not using Dockefiles, it uses your local nav-ui git repository to run yarn directly from it, same for the api and engine. So changes here will be visible on AWS only. Remember to switch off draft mode (PR), otherwise Jenkins is not going to trigger a build.

Check here: https://github.com/fjelltopp/fjelltopp-infrastructure/blob/0019a352e9a37ae4c8da04f3369d2ca0430a88f1/roles/navigator/templates/kubernetes/navigator_ui.yaml#L96

Reply:
Yes! That did it 🎉 

Is there anywhere else I need to make the update?

I've added my new yarn command to:
- navigator-ui/Dockerfile
- fjelltopp-infrastructure/roles/navigator/templates/kubernetes/navigator_ui.yaml

Reply:
that should be enough